src/main/java/com/asana/resources/Workspaces.java

```diff
@@ -1,10 +1,36 @@
 package com.asana.resources;
 
 import com.asana.Client;
+import com.asana.models.*;
+import com.asana.requests.CollectionRequest;
 import com.asana.resources.gen.WorkspacesBase;
 
 public class Workspaces extends WorkspacesBase {
     public Workspaces(Client client) {
         super(client);
     }
+
+    public CollectionRequest<Task> taskTypeahead(String workspace) {
```

Comment:
I suppose this is the best that can be done with this heterogeneous return type. I think I would have preferred to have something like client.tasks.typeaheadInWorkspace for this, but we've already shipped these methods in our other client libraries (and here in Java, incorrectly returning only a Workspace, which is something typeahead never does, 🙄 ). Something to keep in mind for a potential future client.projects.searchInTeam or whatever future search may hold.